lib/hammer_cli_katello/sync_plan.rb

```diff
@@ -31,7 +31,8 @@ class CreateCommand < HammerCLIKatello::CreateCommand
              )
 
       option "--sync-date", "SYNC_DATE",
-             _("start date and time of the synchronization defaults to now"),
+             _("Start date and time for the sync plan." \
+               "Time is optional, if kept blank current system time will be considered"),
```

Comment:
Yeah, sync-date is not optional. We can not create sync plan without sync date. Time part of the syntax  "YYYY-MM-DD HH:MM:SS" is optional, so we can just pass the date "YYYY-MM-DD".

Is this change acceptable ?